dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -1000,3 +1000,20 @@ CWWKD1100.cursor.requires.sort.explanation=Cursor-based pagination requires \
 CWWKD1100.cursor.requires.sort.useraction=Remove the ORDER BY clause from the \
  Query annotation value if one is found there. Add an OrderBy annotation or a \
  repository method parameter to specify the sort criteria.
+
+CWWKD1101.attr.subset.mismatch=CWWKD1101E: The {0} return type of the \
+ {1} method of the {2} repository attempts to return results as the subset \
+ of entity attributes that are represented by the {3} Java record type, \
+ expecting the {4} entity to have attributes named: {5}. To return a subset \
+ of entity attributes as a Java record, ensure that the record component names \
+ are within the set of valid attribute names for the {4} entity: {6}. \
+ Alternatively, use the Select annotation to assign valid entity attribute names. \
```

Comment:
> Alternatively, use the Select annotation to assign valid entity attribute names. 
> To instead return a single entity attribute that has a Java record type, annotate the {1} method to have a Select annotation with its member value assigned to the name of the entity attribute.

Using "Alternatively... instead" here is a bit confusing- are there 2 alternatives or one? ie is "use the Select annotation to assign valid entity attribute names." the same thing as "annotate the {1} method to have a Select annotation..."? Im not sure.

> To return a single entity attribute that has a Java record type, annotate the {1} method to have a Select annotation and assign its member value to the name of the entity attribute.

Reply:
Alternatively, refers to another way "To return a subset of entity attributes as a Java record".
Instead refers to doing something else: "To return a single entity attribute that has a Java record type".

I will remove the word instead and use the language you suggested, which should still be understood by the user.